mentorship-platform/scss/style.scss

```diff
@@ -1,38 +1,80 @@
-$green: green;
-$lime: lime;
-$dark-blue: darkblue;
-$blue: blue;
-$violet: violet;
-$white: white;
-
-* {
-  box-sizing: border-box;
-  margin: 0;
+@@ -12,7 +12,7 @@ $background-color: rgba(52, 205, 205, 0.2);
   padding: 0;
 }
 
+.sections {
+.heading {
+  margin: 30px;
+  padding: 30px;
+  text-align: center;
+@@ -22,6 +22,7 @@ $background-color: rgba(52, 205, 205, 0.2);
 .green {
   background-color: $green;
+  margin: 16px 0;
+  color: $white;
 }
 
 .lime {
-  background-color: $lime;
-}
-
-.darkblue {
-  background-color: $dark-blue;
+@@ -44,15 +45,60 @@ $background-color: rgba(52, 205, 205, 0.2);
+.violet {
+  background-color: $violet;
   margin: 16px 0;
   color: $white;
 }
 
-.blue {
-  background-color: $blue;
-  margin: 16px 0;
+.summarys {
+.sections {
+  margin: 24px 5px;
+  padding: 30;
+  border: 1px solid;
+}
+
+.article {
+  padding: 5px;
+  margin: 20px;
+}
+
+.button {
+  margin: 5px;
+  padding: 5px;
+}
+
+.button--filled {
```

Comment:
lets shorten the css code like this 
`.button {
&--filled {

}
&--outlined {

}
&--active {

}.. and so on
}
`